xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3077,6 +3077,8 @@ platform.livetable.paginationResultsSingle=Result <span class="currentResultsNo"
 platform.livetable.paginationResultsMany=Results <span class="currentResultsNo">{0} - {1}</span> of <span class="totalResultsNo">{2}</span>
 platform.livetable.paginationResults=Results
 platform.livetable.paginationResultsOf=out of
+platform.livetable.false=No
+platform.livetable.true=Yes
```

Comment:
Yes it was my first thinks about it. a new key allow to change livetable value ... but yes/no is generic too :-/.